templates/index.html

```diff
@@ -14,25 +14,46 @@ <h1 class="logo-font">conduit</h1>
           <ul class="nav nav-pills outline-active">
             {% if user.is_authenticated %}
             <li class="nav-item">
-              <a class="nav-link" href="">Your Feed</a>
+              <a
+                class="nav-link {% if active_tab == 'feed' %}active{% endif %}"
+                href="{% url 'index' %}?active_tab=feed"
+                >Your Feed</a
+              >
             </li>
             {% endif %}
             <li class="nav-item">
-              <a class="nav-link active" href="">Global Feed</a>
+              <a
+                class="nav-link {% if active_tab == 'global' %}active{% endif %}"
+                href="{% url 'index' %}"
+                >Global Feed</a
+              >
             </li>
+            {% if tag %}
+            <li class="nav-item">
+              <a class="nav-link {% if active_tab == 'tag' %}active{% endif %}"
+                >{{tag}}</a
+              >
+            </li>
+            {% endif %}
```

Comment:
The `tag` tab isn't actually clickable and just disappears once that filtering goes away.